finsoc/app/assets/javascripts/newsDropdown.js.erb

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
Using .js.erb is ugly and doesn't seem like it's used here? If you don't need the .erb extension, just use .js.